products/sle/main.pm

```diff
@@ -973,15 +977,6 @@ elsif (get_var('HPC')) {
         elsif (check_var('HPC', 'hwloc') && sle_version_at_least('12-SP2')) {
             loadtest 'console/hwloc_testsuite';
         }
```

Comment:
I also believe that whole section:
```
    if (get_var('CLUSTER_NODES')) {
        loadtest 'console/install_all_from_repository' if (get_var('INSTALL_ALL_REPO'));
        loadtest 'console/install_single_package'      if (get_var('PACKAGETOINSTALL'));

        # load hpc multimachine scenario based on value of HPC variable
        # e.g 'hpc/$testsuite_[master|slave].pm'
        my $hpc_mm_scenario = get_var('HPC');
        loadtest "hpc/$hpc_mm_scenario" if $hpc_mm_scenario ne '1';
    }
    else {
        if (check_var('HPC', 'conman')) {
            loadtest 'hpc/conman';
        }
        elsif (check_var('HPC', 'powerman')) {
            loadtest 'console/hostname';
            loadtest 'hpc/powerman';
        }
        elsif (check_var('HPC', 'hwloc') && sle_version_at_least('12-SP2')) {
            loadtest 'console/hwloc_testsuite';
        }
    }
```
can be simplified to:
```
    loadtest 'console/install_all_from_repository' if (get_var('INSTALL_ALL_REPO'));
    loadtest 'console/install_single_package'      if (get_var('PACKAGETOINSTALL'));

    # load any hpc scenario based on value of HPC variable
    # e.g 'hpc/$testsuite.pm'
    my $hpc_scenario = get_var('HPC');
    loadtest "hpc/$hpc_scenario" if $hpc_scenario ne '1';
```
It will work for multimachine tests and single machine as well.